packages/components/src/constants/swipe.jsx

```diff
@@ -0,0 +1 @@
+export const DISABLE_SCROLL_DISTANCE = 10;
```

Comment:
Is it used anywhere besides side_scroller? If not - should be moved to side_scroller folder

Reply:
`DISABLE_SCROLL_DISTANCE` is used two times in /side_scroller/index. We moved it to /side_scroller to have it all in one place.